model/project_ref.go

```diff
@@ -1803,10 +1802,10 @@ func GetProjectSettingsById(projectId string, isRepo bool) (*ProjectSettings, er
 
 // GetProjectSettings returns the ProjectSettings of the given identifier and ProjectRef
 func GetProjectSettings(p *ProjectRef) (*ProjectSettings, error) {
-	hook, err := FindGithubHook(p.Owner, p.Repo)
-	if err != nil {
-		return nil, errors.Wrapf(err, "finding GitHub hook for project '%s'", p.Id)
-	}
+
+	// GetProjectSettings shouldn't error if GitHub app is not set up, or installed on a repo.
```

Comment:
hasApp only returns an error if there's an issue getting the installation token, so this comment isn't quite right. I think we can also drop the function name here.